manager/state/raft/raft.go

```diff
@@ -126,6 +124,8 @@ type Node struct {
 
 	snapshotInProgress chan uint64
 	asyncTasks         sync.WaitGroup
+
+	stopped chan struct{}
```

Comment:
I am actually not sure now. `stopCh` is closed only from `Shutdown` and node can stop without `Shutdown` by `removeRaftCh`. So in this terms, it's quite different because cancels everything way earlier on removing node.
But I don't know if `removeRaftCh` is proper workflow after our talk.